cloud/aws/cli/core.py

```diff
@@ -294,10 +323,16 @@ def unhandled_exception(type, value, traceback):
 
 if __name__ == "__main__":
     sys.excepthook = unhandled_exception
+
     namespace = Collection.from_module(sys.modules[__name__])
+
     integ = Collection.from_module(integration)
     integ.configure({"run": {"env": {"VASTCLOUD_NOTTY": "1"}}})
     namespace.add_collection(integ)
+
+    ct = Collection.from_module(cloudtrail)
+    namespace.add_collection(ct)
+
```

Comment:
This is the only point where the core python file needs to be modified to add a new module